packages/flutter_tools/lib/src/error_messages.dart

```diff
@@ -0,0 +1,177 @@
+import 'base/context.dart';
```

Comment:
Let's put `error_messages.dart` in `lib/src/base/`

Reply:
Done.